utils/generate_version_git_file.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -eu
+
+this_script=$(basename "$0")
+version_file=include/VERSION
+version_git_file=include/VERSION_GIT
+
+if [ ! -f "$version_file" ]; then
+    echo "Error: execute ${this_script} from repository top level."
+    exit 1
+fi
+
+if ! (git log -1 >/dev/null); then
+    echo "Error: git not available."
+    exit 1
+fi
+
+grass_version_git=$(git rev-parse --short HEAD)
+gitdate_utc_local=$(TZ=UTC0 git log -1 --date=iso-local --pretty=format:"%cd" -- include)
+grass_headers_git_date=$(echo "$gitdate_utc_local" \
+    | sed 's/ /T/' | sed 's/ //' | sed 's/\(..\)$/:\1/')
```

Comment:
As you might have noticed I found a new, more platform independent, way of extracting Git commit date in UTC, than presently in : 
https://github.com/OSGeo/grass/blob/4d2dac47893779757585605b70903dc2277ba23c/configure.ac#L172-L175

The `date -d` is a GNU date option not available in BSD systems.

If you find this indeed working, we may replace this in `configure.ac` too.

The current solution in configure.ac is not consistent between GNU and BSD and the latter always use the current date (of configuration).

Reply:
But git now becomes a build dependency as a counterpart (or was it already one)

Reply:
> But git now becomes a build dependency as a counterpart (or was it already one)

No. No change on that side.

Reply:
> If you find this indeed working, we may replace this in configure.ac too.
> 
> The current solution in configure.ac is not consistent between GNU and BSD and the latter always use the current date (of configuration).

It's these comments that prevented me from pushing more or resolving this comment, since it wasn't addressed yet.

Reply:
Tested on Linux (Debian): `2024-02-07T12:39:27+00:00`

Reply:
@lbartoletti May I bother you to test the above bash snippet (executed in a git repository) on a FreeBSD system.

Reply:
I just added a commit with the new commit date code to configure too, with added configure log message. Mac, Ubuntu and the Windows configuration result with identical date.

Reply:
@nilason `2024-02-07T12:39:27+00:00`
Perfect, thanks to removing the `date -d` and for this script.

Reply:
> @nilason `2024-02-07T12:39:27+00:00` Perfect, thanks to removing the `date -d` and for this script.

Great, thanks for checking!